apps/studio-client/apps/main/form-editor-studio-plugin/src/helper/FormElementsManager.ts

```diff
@@ -93,7 +144,7 @@ class FormElementsManager {
     this.#getRootNodeStruct().getType().removeProperty(elementId);
   }
 
-  static #generateRandomId(): number {
+  static generateRandomId(): number {
     return Math.floor(Math.random() * (int.MAX_VALUE - 0 + 1)) + 23;
```

Comment:
Do you doubt the usefulness of the change or do you think, the new line looks strange?
The change was necessary because the Method is used from outside the class: It is considered "private" when prefixed with a "#"